server/odc-service/src/main/java/com/oceanbase/odc/service/connection/table/TableService.java

```diff
@@ -224,4 +258,17 @@ private List<Table> entitiesToModels(Collection<DBObjectEntity> entities, Databa
         return tables;
     }
 
+    private String initDBSupportExternalTableVersion(@NonNull DialectType dialectType) {
```

Comment:
we may add a new method on `VersionDiffConfigService`, such as `VersionDiffConfigService#getSupportFeatures`, btw, the method with this name has been existed, we may create another one with different parameters, just like this:
```java
public List<OBSupport> getSupportFeatures(String version, ConnectType connectType) {
 ...
}
```
please notice the duplicated code, we may reuse some codes with the current method named `getSupportFeatures`